bulbs/api/views.py

```diff
@@ -189,10 +189,10 @@ def trash(self, request, **kwargs):
         doc_type = content.__class__.search_objects.mapping.doc_type
 
         try:
-            Content.search_objects.client.delete(
-                index=index,
-                doc_type=doc_type,
-                id=content.id)
+            self.model.search_objects.client.delete(
+                    index=index,
```

Comment:
This is minor, but best not to introduce non-standard (PEP8) indentation to existing code throughout this file. All whitespace-only changes in this file should be reverted. Someday (soon!) we want a clean lint run on django-bulbs.